File: Sequence/q1-q99/q1-q9/q4/q4.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums1.extend(nums2)
        nums1.sort()
        if (len(nums1) % 2):
            return nums1[int(len(nums1)/2)]/1.0
        return (nums1[int(len(nums1)/2)]+nums1[int(len(nums1)/2-1)]) /2.0

File: q4/q4.cpp
#include <algorithm>
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        int size1 = nums1.size();
        int size2 = nums2.size();
        int my_size = size1+ size2;
        std::vector<double> my_vec;
        for(int i = 0; i < size1; i++){
            
            my_vec.push_back(nums1[i]);
        }
        for(int j = 0; j < size2; j++){
                my_vec.push_back(nums2[j]);
            }
        std::sort(my_vec.begin(), my_vec.end());
        if(my_size%2 == 0)
            return (my_vec[my_size/2]+my_vec[my_size/2-1])/2;
        else
            return my_vec[my_size/2];
    }
};


